### PR TITLE
[blog] linkedin goodbye post

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    I used LinkedIn since 2008 when it was sending spam to all my friends. After 8 years of tuning my profile to 95 % score, reading mostly copy-pasted messages,
+    checking false notifications, flirting with HR girls and hoping for up votes for my recent skills, <strong>I've decided it is time to let it go</strong>. Why and what I'm gonna do now?
+'''
+lang: en
+---
+
+*Disclaimer: even if I get few abroad offers, my experience is related to work environment in the Czech Republic.*
+
+I was reviewing my 2016 on social networks disconnection: I reduced 350 friends on Facebook to 20, unfollowed all but 2 groups and I dropped all people I follow on Twitter. 
+My attention got better and I could focus on things that really mattered to me. That's why I started though about the next step.
```
Comment (paveljanda) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Maybe... My attention got better and I could focus on things that really matter to me. That's why I started thinking about the next step.

And one more thing: You said you "started thinking about the next step" because of you "could focus on things that really matter". Is that really the main cause?

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+And LinkedIn account was something scratching on my mind for a long time.
```
Comment (paveljanda) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: were

Comment (paveljanda) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Maybe ... the LinkedIn account?

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+So I asked myself?
+
+
+## What is LinkedIn good for me?
```
Comment (paveljanda) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Maybe... What is LinkedIn doing for me? Or... What are the advantages of using LinkedIn profile

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+- I got some references and up-votes for my skills, I attended university so I thought it gives me credibility.
+- It gave me self-confidence when talking about money.
+- Everybody did it, so did I (like that time I jumped out of the window :).
+
+
+## Where LinkedIn Fails for me  
+
+### It's kinda Broken and Stuck Service
+
+I was getting fake notifications of new connections. I got over 20 pings for 6 moths-old and answered message.
```
Comment (paveljanda) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: 6 moths-old - why the dash?

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+Also, I didn't notice many changes between 2008 and 2016. It could be much more further. Btw, do you remember MySpace?
```
Comment (paveljanda) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Maybe... It could be so much more.

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+ 
+
+### Irrelevant Messages - 95 % Job Offers are Spam 
+
+I only got **2 jobs I wanted** in the whole 8 years. Most of offers are general random messages from bots or people,
```
Comment (paveljanda) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Maybe... Instead of: "I only got 2 jobs I wanted in the whole 8 years." somethink like: "I received only 2 job offers I wanted in past 8 years."

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+who don't even read descriptions like *part-time only* or *Symfony project only*.
```
Comment (paveljanda) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Maybe... that didn't even read descriptions like *part-time only* or *Symfony project only*.

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+
+
+### Target group Bias
+
+I never look for PHP full-time job. I wanted part-times only. Not really favorite position on LinkedIn, is it?
+last 2 years I only offered consultancy and lecturing. Well, is that a full-time position?
```
Comment (paveljanda) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Capital L

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+
+### Target group Bias
+
+I never look for PHP full-time job. I wanted part-times only. Not really favorite position on LinkedIn, is it?
+last 2 years I only offered consultancy and lecturing. Well, is that a full-time position? 
+
+
+### More info sources = more outdated info  
+
+I put my work information on my website, Facebook and LinkedIn. The more I have, the more outdated they are. And I'm not alone.
```
Comment (paveljanda) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Maybe... the more outdated they were?

**Comment by TomasVotruba** [comment]
Thanks for feedback @paveljanda, much appreciated!

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md (hunk 1 above)
Comment (mahyn89) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: that really matter to me...............started to think.........

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+perex: '''
+    I used LinkedIn since 2008 when it was sending spam to all my friends. After 8 years of tuning my profile to 95 % score, reading mostly copy-pasted messages,
+    checking false notifications, flirting with HR girls and hoping for up votes for my recent skills, <strong>I've decided it is time to let it go</strong>. Why and what I'm gonna do now?
+'''
+lang: en
+---
+
+*Disclaimer: even if I get few abroad offers, my experience is related to work environment in the Czech Republic.*
+
+I was reviewing my 2016 on social networks disconnection: I reduced 350 friends on Facebook to 20, unfollowed all but 2 groups and I dropped all people I follow on Twitter.
```
Comment (mahyn89) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: nerozumím moc větě se spojení "on social networks disconnection"

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md (hunk 2 above)
Comment (mahyn89) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: spíš "was something that had been scratching my mind for a long time" nebo jen jednoduše "LinkedIn account was scratching my mind"

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md (hunk 3 above)
Comment (mahyn89) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Why is LinkedIn good for me? nebo What is LinkedIn good for?

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md (hunk 6 above)
Comment (mahyn89) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: co takhle during the whole 8 years?

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md (hunk 7 above)
Comment (mahyn89) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: I don´t think so

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+
+I would recommend keeping that profile. 
+
+But when you answered *NO* to all of these, here what you could do:
+
+## Be Proactive &ndash; Find Yourself better Job
+
+- Build personal brand on good old relationships.
+- Talk with people.
+- Find active social networks where you interact with you peers - for me it is Facebook or our PHP Comminity [Péhápkaři Slack](http://pehapkari.slack.com).
```
Comment (mahyn89) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: you"r" peers ................Community

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md (hunk 1 above)
Comment (TomasVotruba) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: 1. Good point!

2. You are right, it's rather "That's why I could better focus on quality of my current environment."

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md (hunk 3 above)
Comment (TomasVotruba) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: I know what you mean. Original formulation sounds weird. What about "What Good LinkedIn brings me?".

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md (hunk 6 above)
Comment (TomasVotruba) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Well I got many offers I wanted, bot only 2 of them were both great and realized.

Comment (TomasVotruba) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: I would rephrase it: "Only **2 jobs offers end up in real contract** in the whole 8 years."

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md (hunk 9 above)
Comment (TomasVotruba) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: Yea, it sounds weird.
Rather rephrase: "The more I have, the more outdated they become in time."

File: source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md (hunk 10 above)
Comment (TomasVotruba) [inline source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md]: I rephrased that

**Comment by TomasVotruba** [comment]
Thank you both for making this much better products!
